app/grandchallenge/reader_studies/views.py

```diff
@@ -379,65 +379,23 @@ class ReaderStudyStatistics(
     # If the permission is changed to 'read', we need to filter these values out.
 
 
-class ReaderStudyDisplaySetList(
-    LoginRequiredMixin, ObjectPermissionRequiredMixin, PaginatedTableListView
-):
+class ReaderStudyDisplaySetList(CivSetListView):
     model = DisplaySet
     permission_required = (
-        f"{ReaderStudy._meta.app_label}.change_{ReaderStudy._meta.model_name}"
+        f"{ReaderStudy._meta.app_label}.change_{DisplaySet._meta.model_name}"
```

Comment:
Should be view permission, no?

```suggestion
        f"{ReaderStudy._meta.app_label}.view_{DisplaySet._meta.model_name}"
```

Reply:
Then readers would be able to see the cases, which I don't think we want, do we? At the moment, we check for the change reader study permission, so readers don't get access to this view.

Reply:
Ah good point! That is something to be careful about and is unexpected for a list view. Could you add that as a comment and make sure it is covered with a test?